src/index.ts

```diff
@@ -1,52 +1,33 @@
 require("dotenv");
-import https from "https";
-import fs from "fs";
-import express from "express";
-import { seedDatabase } from "./services/postgres/seed";
-import path from "path";
-import connect from "./services/postgres/connect";
-import AppRouter from "./routes/index";
+import { setup_pool } from "./services/postgres/connect";
+import { baseSetup } from "./server";
+import { sendSMS } from "./services/twilio/sms";
 
-const port = process.env.PORT || 3001;
-const app = express();
-const db = connect.db;
-const appRouter = new AppRouter(app);
-appRouter.route();
-// Serve static documentation files
-app.use(express.static(path.join(__dirname, "/docs")));
-app.route("/").get((req, res) => {
-    res.sendFile(path.join(__dirname + "/../docs/index.html"));
+const db = setup_pool();
+const server = baseSetup(db);
```

Comment:
We need to refactor code eventually, in some places we use camel case, in others we use pothole

Reply:
Ya, it is the same with file names. perhaps a consistent convention should be devised to avoid confusion